specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2024-08-01-preview/cluster.json

```diff
@@ -0,0 +1,2297 @@
+        },
+        "saskey": {
+          "type": "string",
+          "description": "The shared access signature key."
+        },
+        "fileshare": {
+          "type": "string",
+          "description": "The file share name."
+        },
+        "enableSecureChannel": {
+          "type": "boolean",
+          "description": "Enable secure channel or not, it's an optional field. Default value is false when cluster version < 5.1 and true when cluster version >= 5.1 , "
+        }
+      },
+      "description": "The storage Account."
+    },
+    "StorageProfile": {
+      "type": "object",
+      "properties": {
+        "storageaccounts": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StorageAccount"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ],
+          "description": "The list of storage accounts in the cluster."
+        }
+      },
+      "description": "The storage profile."
+    },
+    "ExcludedServicesConfig": {
+      "type": "object",
+      "properties": {
+        "excludedServicesConfigId": {
+          "type": "string",
+          "description": "The config id of excluded services."
+        },
+        "excludedServicesList": {
+          "type": "string",
+          "description": "The list of excluded services."
+        }
+      },
+      "description": "The configuration that services will be excluded when creating cluster."
+    },
+    "NetworkProperties": {
+      "type": "object",
+      "description": "The network properties.",
+      "properties": {
+        "outboundDependenciesManagedType": {
+          "type": "string",
+          "description": "A value to describe how the outbound dependencies of a HDInsight cluster are managed. 'Managed' means that the outbound dependencies are managed by the HDInsight service. 'External' means that the outbound dependencies are managed by a customer specific solution.",
+          "enum": [
+            "Managed",
+            "External"
+          ],
+          "x-ms-enum": {
+            "name": "OutboundDependenciesManagedType",
+            "modelAsString": true
+          }
+        },
+        "resourceProviderConnection": {
+          "type": "string",
+          "description": "The direction for the resource provider connection.",
+          "enum": [
+            "Inbound",
+            "Outbound"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceProviderConnection",
+            "modelAsString": true
+          }
+        },
+        "privateLink": {
+          "type": "string",
+          "description": "Indicates whether or not private link is enabled.",
+          "enum": [
+            "Disabled",
+            "Enabled"
+          ],
+          "x-ms-enum": {
+            "name": "PrivateLink",
+            "modelAsString": true
+          }
+        },
+        "publicIpTag": {
```

Comment:
>publicIpTag [](http://example.com/codeflow?start=9&length=11)

Please update the examples to reflect the usage of this property.

Reply:
Done in `\specification\hdinsight\resource-manager\Microsoft.HDInsight\preview\2023-08-15-preview\examples\CreateHDInsightClusterWithCustomNetworkProperties.json`